features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin_new/source/src/app/components/Throttling/Custom/AddEdit.jsx

```diff
@@ -100,8 +120,22 @@ function AddEdit(props) {
     const [validationError, setValidationError] = useState([]);
     const [editMode, setIsEditMode] = useState(false);
     const restApi = new API();
+    const { policyId } = props.match.params;
 
     useEffect(() => {
+        if (policyId !== 'create') {
+            setIsEditMode(true);
```

Comment:
I think we can set this state at the state initialization time 
i:e
```javascript
const [editMode, setIsEditMode] = useState(policyId !== 'create');
```

Reply:
Fixed with `6317b1b`